eliza-build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+webhook_url="https://hooks.slack.com/services/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"
+
+eval "$@"
+status=$?
+
+message='{"text":"@eliza, your build failed! :cry:"}'
+if [ $status -eq 0 ]; then
+    message='{"text":"@eliza, your tests passed :D"}'
+fi
+curl -X POST -H 'Content-type: application/json' --data "$message" "$webhook_url"
```

Comment:
This file shouldn't be in this PR.

Reply:
`git add .` considered harmful

Reply:
Agh. Sorry. I'll make it leave.

Now you know how the magic happens though, I guess. :P